src/main/java/be/xplore/pricescraper/PriceScraperApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
 
 @SpringBootApplication
+@ConfigurationPropertiesScan("be.xplore.pricescraper.config")
```

Comment:
There aren't any ConfigurationProperties in this project yet, so this seems a bit redundant for now. However, ConfigurationProperties do have advantages over `@Value` injected properties.